### DISCUSSION TITLE
Infinite Refresh Loop @ Homepage (Fresh Install)

### DISCUSSION THREAD

**Discussion by bakirgdev** [pocketbase/pocketbase#5325] [Q&A]
This is my first PocketBase installation. After registration, I encounter the following bug:

https://github.com/user-attachments/assets/e2fc18de-ceb2-44ac-9f73-a6982ee476d0

Here is some, hopefully helpful, environment info:
- using cloudflare
- made specific user for this app purpose (using CloudPanel)
- made reverse proxy to forward localhost:8090 to domain
- added cloudflare origin SSL cert (no Let's Encrypt,...)
- allowed traffic only from CloudFlare
- registration was normal, regular gmail and random strong password used
- other pages seem to work fine

EDITS
- v0.22.18

**Comment by ganigeorgiev** [comment]
I don't think this an issue with PocketBase and it is most likely something with your deployment setup, for example misconfigured reverse proxy but I'm not sure how to help with that.

**Reply by ganigeorgiev** [reply]
It could be also caused by a conflicting browser extension, I'm not sure.

**Reply by bakirgdev** [reply]
I wouldn't post an Issue without checking everything first... And no, it is not the browser/extensions (tested on plain Edge). There is no nuclear science with reverse proxies, just forward a port to domain (in simple terms). CloudFlare should also not be the problem IMO, since it is constantly trying to fetch the favicon and this url `https://api-prijevoz-ba.bakir.dev/api/collections/_pb_users_auth_/records?page=1&perPage=40&sort=-created&skipTotal=1&filter=&expand=&fields=` and is constantly getting 403. Any other advice? @ganigeorgiev

**Reply by ganigeorgiev** [reply]
While I could understand your frustration, there were no major changes to the Admin UI in the recent releases and no one else has reported such issue so far. I'm using PocketBase in production at https://app.presentator.io/ and I'm not able to reproduce it.

While it is not _"nuclear science"_ reverse proxies come with their caveats and misconfigurations are common. Since in the video the hashbang url query parameters are constantly stripped double check if they are forwarded properly.

**Reply by ganigeorgiev** [reply]
For the 403 errors - again make sure that the proxy is configured to forward the `Authorization` header exactly as it was sent.
